soc/nxp/mcx/CMakeLists.txt

```diff
@@ -5,3 +5,6 @@
 #
 
 add_subdirectory(${SOC_SERIES})
+
+# This is for access to pinctrl macros
+zephyr_include_directories(../kinetis/common)
```

Comment:
I don't want to put anything in the family level of MCX still for now (see comments in Kconfig files at family level, I guess I should have put one here as well)

Also, isn't a pinctrl_soc.h file still needed? look at how the kinetis family pinctrl_soc.h is right now, it just includes the common file

Reply:
Updated, thanks a lot. Please help double check.